server/autolinkplugin/plugin.go

```diff
@@ -103,14 +103,14 @@ func (p *Plugin) ProcessPost(c *plugin.Context, post *model.Post) (*model.Post,
 
 			channel, cErr := p.API.GetChannel(post.ChannelId)
 			if cErr != nil {
-				mlog.Error(cErr.Error())
+				p.API.LogError(cErr.Error())
```

Comment:
Would you please give these log Messgesanges a bit more context. Something like `Failed to get channel` using fielded logging.